src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -980,16 +902,18 @@ void ScaledDotProductAttention::createPrimitive() {
     ScaledDotProductAttentionKey key = {rtPrecision};
 
     auto builder = [&](const ScaledDotProductAttentionKey& key) -> std::shared_ptr<Executor> {
-        std::shared_ptr<Executor> executor;
+        std::shared_ptr<Executor> executor = nullptr;
         if (rtPrecision == ov::element::bf16) {
 #ifdef OPENVINO_ARCH_X86_64
             executor = std::make_shared<AttentionExecutor<KT_ONEDNN, ov::bfloat16>>(context);
 #endif
         } else {
 #ifdef OV_CPU_WITH_MLAS
             executor = std::make_shared<AttentionExecutor<KT_MLAS, float>>(context);
-#else
+#elif defined(OPENVINO_ARCH_X86_64)
             executor = std::make_shared<AttentionExecutor<KT_ONEDNN, float>>(context);
```

Comment:
sounds great!

Reply:
@luo-cheng2021 I check the brgemm kernel, it needs at least (avx2_vnni, avx2_vnni_2) to implement copy_a/copy_b. Since current MLAS is used for FP32(AVX2/AVX512), I think AVX2 support of brgemm is not a urgent request. I suggest that we postpone this request.

Reply:
Sure.